src/hooks/handleRequest/Solana.test.ts

```diff
@@ -0,0 +1,93 @@
+import { SOLANA_SIGNING_METHODS } from "@/data/methods/Solana.methods";
+import { handleSolanaRequest } from "./Solana";
+import { Account, WalletAPIClient } from "@ledgerhq/wallet-api-client";
+import type { IWalletKit } from "@reown/walletkit";
+import { vi } from "vitest";
+import * as signTransactionModule from "./signTransaction/signTransaction";
+import * as signMessageModule from "./signMessage/signMessage";
+
+vi.mock("./utils");
+
+describe("Testing Solana request handler mapping", () => {
+  beforeEach(() => vi.clearAllMocks());
+
+  it("should call sign transaction when request method is solana_signTransaction", async () => {
```

Comment:
We are testing the implementation details instead of the expected behaviour given an input we want an output, it shouldn't matter if I call function x or y with other arguments or name if for a given input I still get the expect output